app/views/talks/_assemble_form.html.erb

```diff
@@ -0,0 +1,123 @@
+                        id: 'create-new-talk-medium',
+                        data: { toggle: 'tooltip' },
+                        title: t('buttons.create_medium'),
+                        style: 'text-decoration: none;',
+                        remote: true %>
+            <% if talk.media.present? %>
+              <ul id="sortableTalkMedia">
+                <% talk.media.each do |m| %>
+                  <li>
+                    <%= link_to m.sort_localized + ', ' + m.local_info,
+                                edit_or_show_medium_path(m),
+                                class: textcolor(m),
+                                data: { id: m.id } %>
+                  </li>
+                <% end %>
+              </ul>
+              <%= f.hidden_field :media_order,
+                                 value: talk.media.map(&:id).to_json %>
+            <% else %>
+              <p>
+                <%= t('admin.talk.no_media') %>
+              </p>
+            <% end %>
+          </div>
+        </div>
+      </div>
+    </div>
+    </div>
+    <div class="col-12 col-lg-6">
+      <div class="card bg-light">
+        <div class="card-header bg-generic-card-header">
+          <h5>
+            <%= t('content') %>
+            <%= helpdesk(t('admin.talk.info.description'), false) %>
+          </h5>
+        </div>
+        <div class="card-body">
+          <div class="row mb-3">
+            <div class="col-12">
+              <div class="custom-control custom-checkbox">
+                <%= f.check_box :display_description,
+                                class: 'custom-control-input' %>
+                <%= f.label :display_description,
+                            t('admin.talk.display_description'),
+                            { class: 'custom-control-label' } %>
+                <%= helpdesk(t('admin.talk.info.display_description'),
+                             false) %>
+              </div>
+              <div class="col-12 text-warning ml-2">
+                <% if talk.description.present? && !talk.display_description %>
+                  <span class="badge badge-warning">
+                    <%= t('basics.info') %>
+                  </span>
+                  <%= t('admin.talk.description_warning') %>
+                <% end %>
+              </div>
+            </div>
+          </div>
+          <%= f.trix_editor :description,
+                            class: 'bg-light',
+                            id: 'talk-description-trix',
+                            data: { content: talk.description,
+                                    preview: 'talk-description-preview' } %>
+          <div class="row my-2">
+            <div class="col-12">
+              <%= t('basics.preview') %>
+            </div>
+          </div>
+          <div class="alert alert-dark bg-mdb-color-lighten-6"
+               id="talk-description-preview">
+            <%= raw(talk.description) %>
```

Comment:
should this be really raw or are its content sanitised in any way shape or form? Students might include malicious <script>-tags into their description.

Reply:
HAs been addressed in the latest commit.